PAYMENTS.md

```diff
@@ -0,0 +1,39 @@
+# Integrate your Stripe account to accept your user's payments
+Here you will find all the information that you need to setup your Stripe account into OSEM.
+If you have any problems with installing don't hesitate to [contact us](https://github.com/openSUSE/osem#contact)
+
+## Configure Stripe into the application
+To configure Stripe into your application all you need to do is add the private and publishable keys into the Rails environment.
+
+Add your Stripe API keys in `config/secrets.yml` into these variables:
+
+If you are using the application in development mode your config should look like this:  
+  `stripe_publishable_key = 'pk_**test**_random123example456'`  
+  `stripe_secret_key = 'sk_**test**_random123example456'`
```

Comment:
Hi @rishabhs95, I think we're still missing Chris's point of making a note point to where the users can sign up for Stripe for free.